pyquil/device.py

```diff
@@ -49,89 +46,77 @@ def to_dict(self):
 
         The dictionary representation is of the form::
 
-            {
-                "id": {
-                    "name": "example_qpu",
-                    "version": "0.1",
-                    "timestamp": "23423423"
+            [
+                {
+                    "0": {
+                        "type": "Xhalves",
+                        "dead": False
```

Comment:
This is probably already settled elsewhere but in general it's better to avoid the double negative and do "alive": True

Reply:
AFAIK `dead` was chosen so that the default - `False`, or the key not being present in the dictionary - would represent `alive`. That way alive qubits would not need the key at all, making it easier for a reader to spot dead qubits in the JSON, improving readability.

Reply:
In the example then we should drop the dead: False to improve readability